tests/samples/test_tasks.py

```diff
@@ -215,3 +216,66 @@ async def test_deduplicate_sample_names(
     await task.run()
 
     assert await mongo.samples.find().to_list(None) == snapshot
+
+
+@pytest.mark.parametrize("ready", [[True, False], [False, False]])
```

Comment:
Since `ready[1]` is `False` both times, can't this just be hardcoded in the function body.

Then `ready` can just be `[True, False]`.